lib/block-supports/layout.php

```diff
@@ -557,46 +557,48 @@ function gutenberg_incremental_id_per_prefix( $prefix = '' ) {
 function gutenberg_render_layout_support_flag( $block_content, $block ) {
 	$block_type            = WP_Block_Type_Registry::get_instance()->get_registered( $block['blockName'] );
 	$block_supports_layout = block_has_support( $block_type, array( 'layout' ), false ) || block_has_support( $block_type, array( '__experimentalLayout' ), false );
-	$layout_from_parent    = $block['attrs']['style']['layout']['selfStretch'] ?? null;
+	// If there is any value in style -> layout, the block has a child layout.
+	$layout_from_parent = $block['attrs']['style']['layout'] ?? null;
```

Comment:
Tiny nit re: naming: I know the variable wasn't introduced in this PR, but `$layout_from_parent` makes me think of extracting the layout attribute from the parent. Would it be worth naming this something like `$child_layout_style` to match `$child_layout_declarations`?

Reply:
Yeah that's a good point! It's a tiny change so I'll just add it here.